torch/jit/__init__.py

```diff
@@ -1136,14 +1136,23 @@ def __setattr__(self, attr, value):
 
             if hasattr(self, attr):
                 raise RuntimeError("attempting to re-assign constant '{}'".format(attr))
-            if isinstance(value, ModuleList):
+
+            def conv_module_to_const(module_value):
```

Comment:
Can this be the whole ‘convert’? It’s confusing at first glance with the actual conv modules but they’re

Reply:
what do you mean exactly?